provider-directory/src/test/java/gov/va/api/health/providerdirectory/service/LombokTest.java

```diff
@@ -25,10 +25,10 @@
   public void exerciseLombok() {
     for (Class<?> pojoClass :
         Arrays.asList(
-            PpmsProviderSpecialtiesResponse.class,
-            PpmsProviderSpecialtiesResponse.Value.class,
-            ProviderContacts.class,
-            ProviderContacts.Value.class,
+            ProviderSpecialtiesResponse.class,
```

Comment:
These tests were written to exercise Lombok generated code for code coverage. The `lombok.addLombokGeneratedAnnotation` config should make these unnecessary. Go ahead and delete this file and verify that the coverage doesn't go down.

Reply:
Deleting LombokTest.java drops it from 81% classes, 86% lines to 81% classes, 75% lines.  Opting to leave it in.